tests/Behat/MinkContext.php

```diff
@@ -174,7 +174,7 @@ protected function getLanguageSwitcherOverlay(): NodeElement {
    * @throws \Behat\Mink\Exception\ElementNotFoundException
    */
   protected function getBreadcrumb(): NodeElement {
-    $selector = 'nav.ecl-page-header__breadcrumb';
+    $selector = 'nav.ecl-page-header-core__breadcrumb';
```

Comment:
We're missing this `data-ecl-auto-init="BreadcrumbCore"` from the nav tag: https://ecl-twig-js.netlify.app/ec/?path=/story/components-page-headers-page-header-core--title

Reply:
We have such an attribute when we have more than 3 breadcrumb elements. This logic implemented in the code of the ecl-twig component.